dc-cudami-editor/src/locales/de/translation.json

```diff
@@ -81,6 +94,7 @@
     },
     "newTab": "in neuem Tab",
     "next": "Weiter",
+    "no": "Nein",
```

Comment:
Übrigens: `sameTab` und `newTab` sind zwar in der deutschen Übersetzung da, in der Englischen aber nicht mehr.

Reply:
Danke für den Hinweis, die werden beide gar nicht mehr verwendet, hab sie daher gelöscht.